demos/storybook/.storybook/manager-head.html

```diff
@@ -26,4 +26,11 @@
     .css-cz6gk6 {
         color: #007bc1 !important; /* Blue[500] */
     }
+
+    .css-1lq66z2,
+    .css-106odw5,
+    .css-1j4ctyr,
+    .css-1ypp2lh {
+        background: white !important;
+    }
```

Comment:
this will also need to be replicated in Angular and RN storybooks

Reply:
I have some local troubles with RN storybook so I will go ahead and do that for Angular

Reply:
https://github.com/pxblue/angular-component-library/pull/20